src/mpl/BUILD

```diff
@@ -43,6 +43,7 @@ cc_library(
         "//src/sta:opensta_lib",
         "//src/utl",
         "@boost.random",
+        "@cudd",
```

Comment:
Why is this necessary?  mpl has no direct use of cudd

Reply:
I don't see it as fixed.  I see the same in rcx as well.

Reply:
done.